code/modules/trade/datums/trade_stations_presets/hellcat.dm

```diff
@@ -0,0 +1,58 @@
+/datum/trade_station/fbv_hellcat
+	name_pool = list("FBV 'Hellcat'" = "They are sending message, \"Hello, we escorting SAV 'Caduceus', but while we here we can offer you some our extra supplies.\"")
```

Comment:
Greetings. This is the Hellcat. We're currently escorting the Caduceus and we will be departing the system shortly alongside them. We are willing to depart with some extra supplies to get rid of while we're still here.